python/lsst/ts/vent/controller/controller.py

```diff
@@ -327,3 +403,41 @@ def vent_state(self, vent_number: int) -> VentGateState:
                 return VentGateState.CLOSED
             case _:
                 return VentGateState.FAULT
+
+    def getOptoCh(self, *args, **kwargs) -> int:
+        """Calls megaind.getOptoCh or a simulated getOptoCh depending
+        whether the class was instantiated with simulate = True.
+
+        Raises
+        ------
+        ModuleNotFoundError
+            If the megaind module has not been installed, in which case the
+            daughterboard cannot be controlled.
+        """
+
+        assert self.connected
+        if self.simulate:
+            return self.simulator.getOptoCh(*args, **kwargs)
+        else:
+            if not MEGAIND_AVAILABLE:
+                raise ModuleNotFoundError("The megaind module is not available.")
+            return megaind.getOptoCh(*args, **kwargs)
+
+    def setOd(self, *args, **kwargs) -> None:
```

Comment:
Methods should be `snake_case` but, same as above, we need to talk about the use of `megaind`.